src/Google.Cloud.Functions.ConformanceTests/ConcurrentHttpFunction.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright 2022, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Functions.Framework;
+using Microsoft.AspNetCore.Http;
+using System.IO;
+using System.Threading.Tasks;
+
+// Note: no namespace, to make it simpler to run as part of conformance testing
+public class ConcurrentHttpFunction : IHttpFunction
+{
+    public async Task HandleAsync(HttpContext context)
```

Comment:
This is fine - a simpler implementation would just be:

```csharp
public Task HandleAsync(HttpContext context) => Task.Delay(1000);
```

Either is fine though.